challenge-06/challenge-06.js

```diff
@@ -32,7 +35,10 @@ Crie uma função chamada `showTeamPosition` com as seguintes características:
     - Se não houver time para a posição passada, deve retornar a mensagem:
     "Não temos a informação do time que está nessa posição."
 */
-// ?
+function showTeamPosition(qualification) {
+    qualification >5 ? console.log('Não temos a informação do time que está nessa posição.'):
```

Comment:
-1
Use ternários somente quando for fazer o retorno de alguma função ou atribuir o valor do resultado do ternário a uma variável. Fora isso, use o `if`.